Sources/NIOSSH/Key Exchange/SSHKeyExchangeResult.swift

```diff
@@ -46,18 +46,18 @@ extension KeyExchangeResult: Equatable {}
 /// Of these types, the encryption keys and the MAC keys are intended to be secret, and so
 /// we store them in the `SymmetricKey` types. The IVs do not need to be secret, and so are
 /// stored in regular heap buffers.
-struct NIOSSHSessionKeys {
-    var initialInboundIV: [UInt8]
+public struct NIOSSHSessionKeys {
```

Comment:
Can we add a `public init` as well? There's no good reason to prevent users from constructing one of these.

Reply:
done